core/src/main/java/io/questdb/griffin/SqlKeywords.java

```diff
@@ -1019,6 +1019,22 @@ public static boolean isTimestampKeyword(CharSequence tok) {
                 && (tok.charAt(i) | 32) == 'p';
     }
 
+    public static boolean isRegclassKeyword(CharSequence tok) {
+        if (tok.length() != 8) {
+            return false;
+        }
+
+        int i = 0;
+        return (tok.charAt(i++) | 32) == 'r'
```

Comment:
What's SQL example of this? I don't see a test with this query

Reply:
test is hard to see because it is HEX:

```testPythonInsertSelectHex```

If you run this test and debug with breakpoint inside function in question you should see usage